src/Microsoft.AspNet.Mvc.Core/MvcOptions.cs

```diff
@@ -71,12 +71,19 @@ public AntiForgeryOptions AntiForgeryOptions
 
         /// <summary>
         /// Gets a list of <see cref="ExcludeValidationDescriptor"/> which are used to construct a list
-        /// of exclude filters by <see cref="IValidationExcludeFiltersProvider"/>,
+        /// of exclude filters by <see cref="IValidationExcludeFiltersProvider"/>.
         /// </summary>
         public List<ExcludeValidationDescriptor> ValidationExcludeFilters { get; }
             = new List<ExcludeValidationDescriptor>();
 
         /// <summary>
+        /// Gets a list of <see cref="CacheProfile"/> which are pre-defined settings for
+        /// <see cref="ResponseCacheFilter"/>.
+        /// </summary>
+        public List<CacheProfile> CacheProfiles { get; }
```

Comment:
Make this a dictionary. and remove name from CacheProfile. It will validate no duplicates in the input and remove the unnecessary `Name` property from the input.